### ISSUE TITLE
Preflight check has absurd requirement of swap

### ISSUE THREAD

**Issue by HalJordan** [starcitizen-lug/lug-helper#51]
Passed Checks:
- Wine is installed on your system.
- Winetricks is installed and up to date.
- Your system has 31.3  GB of memory.
- Your CPU supports the necessary AVX instruction set.
- vm.max_map_count is set to 16777216.
- Hard open file descriptors limit is set to 1048576.

Failed Checks:
- You don't appear to have swap space configured.
    We recommend configuring an 8-16 GB swap file.

Swap is dead. We have access to enough memory nowadays, it's no longer necessary, and it's HIGHLY detrimental to the lifespan of SSD based storage which is fast becoming the norm if it hasn't already. I get this being a suggestion if sysmem < 8 GB, but should not be a standard recommendation regardless of the amount of system memory.

**Comment by the-sane** [comment]
Thanks for the suggestion! I’m pretty sure Star Citizen won’t even run on a system with 8GB of RAM. We still have plenty of Penguins using systems with 16GB RAM and swap is absolutely required for them. The game will easily chew through 32GB RAM on extended play sessions and then crash when it runs out. We’ve even had some Penguins with 64GB RAM experiencing unexplainable crashes until they configured some kind of swap space. 

In short, it still seems to be required for a lot of people right now so, unfortunately, it’s not yet dead.
